packages/components/src/navigable-container/container.tsx

```diff
@@ -98,9 +116,11 @@ class NavigableContainer extends Component {
 			// from scrolling. The preventDefault also prevents Voiceover from
 			// 'handling' the event, as voiceover will try to use arrow keys
 			// for highlighting text.
-			const targetRole = event.target.getAttribute( 'role' );
+			const targetRole = ( event.target as HTMLDivElement ).getAttribute(
+				'role'
+			);
```

Comment:
_Technically_, `event.target` can be `null`, which means that calling `getAttribute` would error.

Maybe we could change the code to something like

```suggestion
			const targetRole = (
				event.target as HTMLDivElement | null
			 )?.getAttribute( 'role' );
```

Reply:
Good idea, committed in https://github.com/WordPress/gutenberg/pull/49377/commits/29aef980be8a664c467bc0dc94199e35965a534a